core/observability/metrics.py

```
@@ -0,0 +1,39 @@
+
+        out_count = await redis_pool.scard('out_sms_numbers')
+        OUT_SMS_NUMBERS_COUNT.set(out_count)
+
+        blacklist_count = await redis_pool.scard('blacklist_mobiles')
+        BLACKLIST_MOBILES_COUNT.set(blacklist_count)
+
+        # Count abuse keys using scan (returns all matching keys)
+        abuse_keys = await redis_pool.scan('abuse_counter:*')
+        ABUSE_COUNTER_KEYS.set(len(abuse_keys))
+
+    except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)